src/components/safe-apps/AddCustomSafeAppCard/AddCustomSafeAppCard.tsx

```diff
@@ -46,4 +44,4 @@ const AddCustomAppCard = ({ onSave, safeAppList }: Props) => {
   )
 }
 
-export { AddCustomAppCard }
+export default AddCustomSafeAppCard
```

Comment:
not a big fan of named > default export change: https://basarat.gitbook.io/typescript/main-1/defaultisbad

no need to change, just sharing why it was done the way it was done before